dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/datax/DataxTask.java

```diff
@@ -554,7 +554,7 @@ public String loadJvmEnv(DataxParameters dataXParameters) {
         sql = sql.replace(";", "");
 
         try (
-                Connection connection = DatasourceUtil.getConnection(DbType.valueOf(dataXParameters.getDtType()), baseDataSource);
+                Connection connection = DatasourceUtil.getConnection(DbType.valueOf(dataXParameters.getDsType()), baseDataSource);
```

Comment:
Can we modify the method 
```java
public String[] tryExecuteSqlResolveColumnNames(BaseConnectionParam baseDataSource, String sql)
``` 
to 
```java
public String[] tryExecuteSqlResolveColumnNames(DbType dbType, BaseConnectionParam baseDataSource, String sql)
```

Reply:
@ruanwenjun I agree with what you said, the code has been updated.